### PR TITLE
Adding a few settings required for spreadsheet bulk upload.

### PR DESCRIPTION
**Pull request by tingulfsen**
These are just a few new settings we need to have for the bulk upload stuff to work.

### CODE REVIEW

File: config/environments/test.rb [hunk 1]
```diff
@@ -60,8 +60,9 @@
       spreadsheet           Settings.SPREADSHEET_URL
     end
     bulk_metadata_directory Settings.BULK_METADATA.DIRECTORY # Directory for storing bulk metadata job info
-    bulk_metadata_log       Settings.BULK_METADATA.LOG # Bulk metadata log file
-    bulk_metadata_xml       Settings.BULK_METADATA.XML # Bulk metadata XML output file
+    bulk_metadata_log       Settings.BULK_METADATA.LOG       # Bulk metadata log file
```
Comment (mejackreed) [inline config/environments/test.rb]: Are these required for the tests to run?

**Comment by mejackreed** [comment]
Could these two commits be squashed into one?

**Comment by tingulfsen** [comment]
Yeah, I can squash them. Come to think of it, none of the bulk_metadata variables are required for the tests to run, so I can remove that stuff and squash everything into a single commit if that's preferred.

**Comment by mejackreed** [comment]
Sounds good!

**Comment by jmartin-sul** [comment]
@tingulfsen, re: "none of the bulk_metadata variables are required for the tests to run"

is it that the bulk metadata variables aren't needed by the tests now that integration tests are skipped (PR #53)?  specifically adding those parameters helped get rid of some failures earlier this week (see https://github.com/sul-dlss/argo/commit/85a002a64fd7265dbb748acedf87413e9a265d3f).

though, as per a message i just posted in the argo slack channel, i think what we actually want to do is incorporate the changes from the second commit in this PR, and then get rid of test.rb.example (which would require getting rid of the reference to it in .travis.yml).  even if PR #53 made those settings moot in test.rb, i think we still want to get rid of test.rb.example (via a different PR than this one).

**Comment by jmartin-sul** [comment]
:+1: seems like the right thing, tests look good

**Pull request merged**